tailcfg/tailcfg.go

```diff
@@ -138,9 +138,10 @@ func (emptyStructJSONSlice) MarshalJSON() ([]byte, error) {
 func (emptyStructJSONSlice) UnmarshalJSON([]byte) error { return nil }
 
 type Node struct {
-	ID       NodeID
-	StableID StableNodeID
-	Name     string // DNS
+	ID         NodeID
+	StableID   StableNodeID
+	Name       string   // DNS
+	OtherNames []string // extra DNS names
```

Comment:
I'd prefer this not be on Node but rather be an extra set of records on DNSConfig instead.

I'll send a PR draft as an example.

Reply:
Sent https://github.com/tailscale/tailscale/pull/2150 for discussion